include/revng/Storage/Path.h

```diff
@@ -58,9 +58,9 @@ class FilePath : public PathBase {
   static FilePath fromLocalStorage(llvm::StringRef Path);
 
   /// This function will return another FilePath with the extension added
-  FilePath addExtension(llvm::StringRef Extension) const {
+  std::optional<FilePath> addExtension(llvm::StringRef Extension) const {
```

Comment:
Doxygen: in which cases we return `std::nullopt`?

Reply:
Fixed doxygen